config/zones.json

```diff
@@ -2695,8 +2695,31 @@
     ]
   },
   "IT-SAR": {
+    "bounding_box": [
+      [
+        7.633799675000148,
+        38.36432526200005
+      ],
+      [
+        10.326670769000089,
+        41.76264069200013
+      ]
+    ],
+    "capacity": {
+      "biomass": 1104,
+      "coal": 979, 
+      "geothermal": 0,
+      "hydro": 154,
+      "hydro-storage": 312,
+      "nuclear": 0,
+      "oil": 238,
+      "solar": 787,
+      "wind": 1054,
+      "unknown": 703
```

Comment:
On the website you link to, I can only find thermal, hydro, geothermal, wind and solar capacity. Could you explain where these numbers comes from?

Reply:
I took the hydro, solar and wind and biomass numbers from Terna, then I used the Entsoe "installed capacity per production unit" source to find the capacity of the installed coal/oil/gas and any unknown capacity, I then also used the hydro capacity from Terna and took the hydro-storage from entsoe (taking the storage capacity out of the given hydro capacity) to get both hydro and hydro-storage.